src/main/groovy/io/saagie/plugin/dataops/utils/directory/FolderGenerator.groovy

```diff
@@ -289,6 +292,62 @@ class FolderGenerator {
 		}
 	}
 	
+	void generateFolderForVariable( ExportVariables exportVariables ) {
+		
+		def variableId = exportVariables.variableEnvironmentDTO.variableDetail.id
+		def urlVariableIdFolder = "${inputDire}${sl}${name}${sl}Variable${sl}${variableId}"
+		def folder = new File(urlVariableIdFolder) ;
+		
+		
+		if (exportVariables.exists()) {
+			def createFolderForVariable = folder.mkdirs()
```

Comment:
You're creating a folder here, does it have to be deleted after?

Reply:
Yes it is handled by the previous export function ( it delete the directory container after zipping the parents)